examples/semantic_search/cohere_wikipedia_search.ipynb

```diff
@@ -0,0 +1,516 @@
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "17a67a1c7524475c885c313e4ea55572",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Combobox(value='', continuous_update=False, options=('What is the answer to the great question of life, the un…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "2f621df55aa64b37a37f1db9f857ea4b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Output()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from ipywidgets import Combobox, Output, Button\n",
+    "from IPython.display import display_markdown, Markdown, display, clear_output\n",
+    "\n",
+    "queries = [\n",
+    "\"What is the answer to the great question of life, the universe, and everything?\",\n",
+    "\"What is the largest city in France\",\n",
+    " \"What is the queen of citadels in Lille?\",\n",
+    "\"What is the Elephant's Graveyard?\",\n",
+    "\"What can you see along the A61 in England?\",\n",
+    "\"What person has a connection with the city of Leicester?\",\n",
+    "\"What are the ancient wards of the city of London?\",\n",
+    "\"Who created mini marvels?\",\n",
+    "\"What happens at the end of Moby Dick?\",\n",
+    "\"What is the Euclidean Algorithm used for?\",\n",
+    "\"Can I use an EV to power my house?\",\n",
+    "\"Who is the main character in The Glass Castle?\",\n",
+    "\"What did Winston Smith do for work?\",\n",
+    "\"What schools are in Great Bend, Kansas?\",\n",
+    "\"Who was the first Catholic nominee for U.S. President for a major party?\",\n",
+    "\"What are the inside and outside styles of the Azerbaijan State Philharmonic Hall?\",\n",
+    "\"What were the sides in the Mali War?\",\n",
+    "\"What is the population of Kingston, New Hampshire?\",\n",
+    "\"What did Jacques Ellul say about mass media communication?\",\n",
+    "\"Who played Lucas Sinclair?\",\n",
+    "\"What is stannic oxide used for?\",\n",
+    "]\n",
+    "\n",
+    "def handler(event):\n",
+    "    if event.name != 'value' or input.value == \"\":\n",
+    "        return\n",
+    "        \n",
+    "    user_query = input.value\n",
+    "    input.value = \"\"\n",
+    "\n",
+    "    with output:\n",
+    "        clear_output()\n",
+    "        run_query(user_query)\n",
+    "    \n",
+    "def run_query(user_query):\n",
+    "    display(Markdown(f\"### User Query\\n{user_query}\"))\n",
+    "        \n",
+    "    nonrag_answer = plain_chain.invoke(user_query)\n",
+    "    display(Markdown(f\"### Non-RAG Answer\\n{nonrag_answer}\"))\n",
+    "\n",
+    "    rag_answer = rag_chain_with_source.invoke(user_query)\n",
+    "    display(Markdown(\"\\n\".join([\n",
```

Comment:
How about putting the answer as well?

Reply:
I'm not sure what you mean.  Do you want me to include human-generated answers for these questions?